RoktDemo/UI/Layout Demo/LayoutDemoView.swift

```diff
@@ -47,17 +47,18 @@ struct LayoutDemoView: View {
                         }
                         
                         switch viewModel.uiState {
-                        case .hasData, .done:
-                            Button(action: {
-                                viewModel.uiState = .hasData
-                            }) {
-                                Text("Refresh preview")
-                            }
-                            .padding(.top)
-                            .buttonStyle(ButtonDefaultOutlined())
+                        // TODO: Re-enable the refresh button once layoutVariantIds are stablised
+                        // case .hasData, .done:
+                        //     Button(action: {
+                        //         viewModel.uiState = .hasData
+                        //     }) {
+                        //         Text("Refresh preview")
+                        //     }
+                        //     .padding(.top)
+                        //     .buttonStyle(ButtonDefaultOutlined())
                             
-                            roktEmbedded
```

Comment:
@honghu-2h  have you tested the embedded layout?

Reply:
@danial-rokt , thanks for pointing out, added back roktEmbedded, only commented out the refresh button.
All 3 styles overlay, embedded and bottomSheet are tested now.